docs/Using-Pantheon/Events-and-Logs.md

```diff
@@ -0,0 +1,194 @@
+    ```
+
+## Event Parameters
+
+Up to three event parameters can have the `indexed` attribute. Indexed parameters are stored as `topics` 
+in the logs. Indexed parameters can be searched and filtered.
+
+Topics are 32 bytes. If an indexed argument is an array (including `string` and `byte` datatypes), 
+the keccak-256 hash of the paramater is stored as a topic. 
+
+Non-indexed parameters are included in the logs `data` but cannot be easily searched or filtered. 
+
+!!! example 
+
+    A Solidity contract that stores one indexed and one non-indexed parameter and has an event that emits the value of each parameter: 
+
+    ```solidity
+    pragma solidity ^0.5.1;
+    contract Storage {
+	  uint256 public valueIndexed;
+	  uint256 public valueNotIndexed;
+
+	  event Event1(uint256 indexed valueIndexed, uint256 valueNotIndexed);
+
+	  function setValue(uint256 _valueIndexed, uint256 _valueNotIndexed) public {
+    	 valueIndexed = _valueIndexed;
+    	 valueNotIndexed = _valueNotIndexed;
+    	 emit Event1(_valueIndexed, _valueNotIndexed); 
+	  }
+    }
+    ```
+
+!!! example   
+    Log entry created by invoking the contract above with `valueIndexed` set to 5 and `valueNotIndexed` set to 7:  
+
+    ```json
+     {
+       "logIndex": "0x0",
+       "removed": false,
+       "blockNumber": "0x4d6",
+       "blockHash": "0x7d0ac7c12ac9f622d346d444c7e0fa4dda8d4ed90de80d6a28814613a4884a67",
+       "transactionHash": "0xe994022ada94371ace00c4e1e20663a01437846ced02f18b3f3afec827002781",
+       "transactionIndex": "0x0",
+       "address": "0x43d1f9096674b5722d359b6402381816d5b22f28",
+       "data": "0x0000000000000000000000000000000000000000000000000000000000000007",
+       "topics": [
+        "0xd3610b1c54575b7f4f0dc03d210b8ac55624ae007679b7a928a4f25a709331a8",
+        "0x0000000000000000000000000000000000000000000000000000000000000005"
+       ]
+     }
+    ```
+
+## Event Signature Hash
+
+The first topic in a log entry is always the the event signature hash. The event signature hash is a keccak-256
+hash of the event name and input argument types. Argument names are ignored. For example, the event `Hello(uint256 worldId)` 
+has the signature hash `keccak('Hello(uint256)'`. The signature identifies to which event log topics belong. 
+
+!!! example
+    
+    A Solidity contract with two different events: 
+
+    ``` solidity	
+	     pragma solidity ^0.5.1;
+         contract Storage {
+  	     uint256 public valueA;
+         uint256 public valueB;
+  
+  	     event Event1(uint256 indexed valueA);
+  	     event Event2(uint256 indexed valueB);
+  
+  	     function setValue(uint256 _valueA) public {
+      	    valueA = _valueA;
+      	    emit Event1(_valueA); 
+  	     }
+  	
+  	     function setValueAgain(uint256 _valueB) public {
+      	    valueB = _valueB;
+      	    emit Event2(_valueB); 
+  	     }
+       } 
+    ```
+
+The event signature hash for event 1 is `keccak('Event1(uint256))` and the event signature hash for event 
```

Comment:
don't you have to add a second single quote at the end like `keccak('Event1(uint256)')` ?